src/utils/config.json

```diff
@@ -0,0 +1 @@
+{"version":"@passageidentity/passage-node 0.0.1"}
```

Comment:
will this be auto updated on a deploy?

Reply:
Yes, there's a step in the Github Action here: https://github.com/passageidentity/passage-node/blob/3cc62954dcb141dba220f1f01087330c75607ec9/.github/workflows/deploy.yml#L41 that updates the version with: https://github.com/passageidentity/passage-node/blob/3cc62954dcb141dba220f1f01087330c75607ec9/package.json#L50